Added a section azureVM

MasterAir opened the pull request:
This section describes how to setup a VM on the azure sandbox. This enables the learner to try the DB tools if they cannot establish connection to the SQL server from within the corporate network.

MasterAir committed:
Added a section azureVM

This section describes how to setup a VM on the azure sandbox. This enables the learner to try the DB
tools if they cannot establish connection to the SQL server from within the corporate network.

priyeshkapadia commented on README.md:
```diff
@@ -626,6 +627,31 @@ Below are the steps to create a basic data architectureℹ️ of a SQL server an
+
+### 💻9️⃣ Setup a windows server VM in your Azure sandbox
+
+Due to the way that the organisation's networking is setup it is not possible to connect directly to SQL servers outside of the corporate network. If using a work computer, you need to setup a virtual machine to install the tools in the next section on. That way the instructions in [section 10](#tools) will work as written.
```
I think it would be useful to give a brief explanation of why this is needed. Suggest adding: ", as the connection is being blocked by the companies firewall" after ... the corporate network.

Then at the end suggest adding: "The virtual machine that is being setup in the instructions below is a server that is running the windows operating system. you will then be able to logon to this server and use it in a similar way as you would use your laptop. It will be setup with the windows operating system so you can install any windows compatible software, e.g VS code, SQL Server Manager and Azure Data Studio. You will also have administrator rights so you can perform the installation yourself. This server doesn't have the firewall restrictions that your laptop has, so it is able to connect to the database that is created in the previous section. The instructions below guide you through how to setup the server and then how to remotely access the server"

MasterAir replied:
Added recommended updates

priyeshkapadia commented on README.md:
```diff
@@ -626,6 +627,31 @@ Below are the steps to create a basic data architectureℹ️ of a SQL server an
+
+1. In your Azure sandbox that you set up in the previous section. Return to the [home](https://portal.azure.com/home) screen and click the Virtual Machines icon in the azure services section at the top of the page.
+
+2. In the Virtual Machines page click the blue Create icon.
```
Add: , then "Azure virtual machine" in the popup menu.

priyeshkapadia commented on README.md:
```diff
@@ -626,6 +627,31 @@ Below are the steps to create a basic data architectureℹ️ of a SQL server an
+
+All the other default options are fine. 
+
+7. Once the VM is deployed you can connect to it using Remote Desktop Protocol - as we did in the morning session. Select the Public IP address from the Azure Sandbox for the computer's name in RDP.
```
Add: "To do this use the "Remote Desktop Connection" application on your own windows laptop." before - as we did in the morning session.

priyeshkapadia reviewed (approved):
I've followed the instructions and they correctly launch a windows server VM.

I've added some suggestions for additional text to provide further explanation of why these steps are being carried out.

MasterAir committed:
Added a little more detail on Azure VM

Added a couple of notes as to why we need to do this.
Added a couple of additional details on steps to create an Azure VM.
